main.go

```diff
@@ -370,3 +382,14 @@ func main() {
 			"\n  mean:\t\t", time.Duration(result.Latency.Mean()))
 	}
 }
+
+func getHostSelectionPolicy(hostSelectionPolicy string, hosts []string) gocql.HostSelectionPolicy {
+	switch hostSelectionPolicy {
+	case "round-robin":
+		return gocql.RoundRobinHostPolicy()
+	case "host-pool":
+		return gocql.HostPoolHostPolicy(hostpool.New(hosts))
+	default:
```

Comment:
Default should report an error, missing case for token aware.

Reply:
sure